src/quo2/components/wallet/wallet_overview/style.cljs

```diff
@@ -4,7 +4,6 @@
 (def container-info
   {:padding-horizontal 20
    :padding-top        12
-   :flex-grow          1
```

Comment:
this is definitely safe? 🤔

Reply:
Did you check if it looks fine in `quo2` and if used in other screens?

Reply:
No, it doesn't look fine, thanks!
That flex-grow was causing problems on wallet accounts page since it is used on the same page with `flat-list`. And when there are not enough items in a flat-list, wallet-overview grows in width and that results in jumps when user switches between "assets" and "collectibles" tabs. 
I will return `flex-grow` and will put `wallet-overview` in an enclosing view of fixed height. It is okay when it grows horizontally, but vertical growth isn't really needed for this component.

Reply:
I think we shouldn't be using fixed height. Most of the time `flex-box` is sufficient